platform/darwin/src/MGLFeature.mm

```diff
@@ -224,3 +286,23 @@ static CLLocationCoordinate2D toLocationCoordinate2D(const mbgl::Point<T> &point
     }
     return shapes;
 }
+
+mbgl::Feature mbglFeature(mbgl::Feature feature, id identifier, NSDictionary *attributes)
+{
+    if (identifier) {
+        NSExpression *identifierExpression = [NSExpression expressionForConstantValue:identifier];
+        feature.id = [identifierExpression mgl_featureIdentifier];
+    }
+    feature.properties = [attributes mbgl_propertyMap];
+    return feature;
+}
+
+NS_DICTIONARY_OF(NSString *, id) *NSDictionaryFeatureForGeometry(NSDictionary *geometry, NSDictionary *attributes, id identifier) {
+    NSMutableDictionary *feature = [@{@"type": @"Feature",
+                                      @"properties": (attributes) ?: [NSNull null],
+                                      @"geometry": geometry} mutableCopy];
+    if (identifier) {
```

Comment:
Sorry, I was in `mbglFeature()` not `NSDictionaryFeatureForGeometry()`. Fixed in acf792b